docs/architecture/architecture.md

```diff
@@ -6,3 +6,7 @@ See the following architecture diagrams, following the [C4 model](https://c4mode
 - [Component](C4Component.md)
 
 These diagrams are made using [Mermaid](https://mermaid.js.org/syntax/c4.html). To view the diagrams in your IDE, install a plugin such as Markdown Preview Mermaid Support for VSCode. Otherwise, you can view the diagrams by pasting the Mermaid code [here](https://mermaid.live/).
+
+# Database
+
+This project uses a CosmosDB database, and data is encrypted at rest.
```

Comment:
(Not important but) It might be worth adding a reference to [here](https://learn.microsoft.com/en-us/azure/cosmos-db/database-encryption-at-rest).